pkg/cloud/retry_manager.go

```diff
@@ -0,0 +1,54 @@
+package cloud
+
+import (
+	"github.com/aws/aws-sdk-go-v2/aws/retry"
+	"time"
+)
+
+const (
+	// retryMaxAttempt sets max number of EC2 API call attempts.
+	// Set high enough to ensure default sidecar timeout will cancel context long before we stop retrying.
+	retryMaxAttempt = 50
+	// retryMaxBackoffAdaptive sets max backoff delay between retry attempts.
+	// Set more aggressive than default of 20 seconds due to adaptive client-side throttling being enabled.
```

Comment:
I'm not sure I understand the reasoning for this - we're lowering the default delay that AWS picked for the adaptive retryer, correct?

Reply:
The SDK has a default 20 second max-backoff for all services. It is the same for default retryer (standard) and adaptive. I can stick with default 20s.